sklearn/neighbors/approximate.py

```diff
@@ -217,6 +217,10 @@ def __init__(self, n_estimators=10, radius=1.0, n_candidates=50,
         self.min_hash_match = min_hash_match
         self.radius_cutoff_ratio = radius_cutoff_ratio
 
+        warnings.warn("LSHForest has poor performance.")
```

Comment:
The original issue mentioned 

> LSHForest should be deprecated and scheduled for removal in 0.21. It should also warn about having bad performance

I could make the deprecation warning: 

        LSHForest has poor performance and has been deprecated in 0.19. It will be removed in version 0.21.